parsec/parsec_internal.h

```diff
@@ -570,9 +570,19 @@ extern int device_delegate_begin, device_delegate_end;
     if(!!(COND)) {                                           \
         PARSEC_TASK_PROF_TRACE((PROFILE), (KEY), (TASK));     \
     }
+#define PARSEC_TASK_PROF_TRACE_FLAGS(PROFILE, KEY, TASK, FLAGS)         \
+    PARSEC_PROFILING_TRACE_FLAGS((PROFILE),                             \
+                           (KEY),                                       \
+                           (TASK)->task_class->key_functions->          \
+                           key_hash((TASK)->task_class->make_key(       \
+                              (TASK)->taskpool, (TASK)->locals ), NULL), \
+                           (TASK)->taskpool->taskpool_id,               \
+                           (void*)&(TASK)->prof_info,                   \
+                           (FLAGS)); 
 #else
 #define PARSEC_TASK_PROF_TRACE(PROFILE, KEY, TASK)
 #define PARSEC_TASK_PROF_TRACE_IF(COND, PROFILE, KEY, TASK)
+#define PARSEC_TASK_PROF_TRACE_FLAGS(PROFILE, KEY, TASK)
```

Comment:
Does not follow the prototype of the other case (#if defined(PARSEC_PROF_TRACE))